src/main/java/tc/oc/pgm/commands/ReportCommands.java

```diff
@@ -0,0 +1,51 @@
+package tc.oc.pgm.commands;
+
+import app.ashcon.intake.Command;
+import app.ashcon.intake.parametric.annotation.Text;
+import net.md_5.bungee.api.ChatColor;
+import org.bukkit.entity.Player;
+import tc.oc.component.Component;
+import tc.oc.component.types.PersonalizedText;
+import tc.oc.named.NameStyle;
+import tc.oc.pgm.api.Permissions;
+import tc.oc.pgm.api.match.Match;
+import tc.oc.pgm.api.player.MatchPlayer;
+import tc.oc.pgm.events.PlayerReportEvent;
+import tc.oc.util.components.Components;
+
+public class ReportCommands {
```

Comment:
I think this would fit better as just ModerationCommands, for when other commands are added

Reply:
Renamed